src/stability_sdk/animation.py

```diff
@@ -219,6 +220,17 @@ def save_settings(self, filename: str):
             save_dict['negative_prompt_weight'] = self.negative_prompt_weight
             json.dump(save_dict, f, ensure_ascii=False, indent=4)
 
+    def prepare_init_image(self, fpath=None):
+        if fpath is None:
+            #fpath = self.args.get('init_image')
+            fpath =  self.args.init_image
+        if not fpath:
+            return
+        img = Image.open(fpath)
+        self.args.height, self.args.width = img.size[0], img.size[1]
+        p = self.prior_frames
+        self.prior_frames = [img, img] + p 
```

Comment:
wait this can't work unless I'm missing something 🤔 the prior frames and other image state is kept as numpy.ndarray in OpenCV format BGR. can do img = cv2.imread(fpath) above to load it into ndarray instead of the `Image.open` call. 
i think the should also be `[img, img]` without the `+p`. there should just be two prior frames so in the case of having init image *and* video init the init image will override the first video frame.

Reply:
Interesting. Bill's already playing with a version of this on the `.bill` branch and doesn't seem to be encountering any issues. You're saying you think this should cause issues because the channel order is permuted relative to what it would have been if the code had used `cv2.imread`?

Reply:
oh I see it works because `image_to_prompt` is handling both cv2 ndarray and PIL Image. the rest of code assumes prior_frames are ndarray for example with `image_mix`